### DISCUSSION TITLE
How to use in create-react-app

### DISCUSSION THREAD

**Discussion by wangfengyuan** [unocss/unocss#1781] [Q&A]
add plugin 
```
 UnoCSS({
          presets: [
            presetAttributify(),
            presetUno(),
            presetIcons({
              scale: 1.2,
              cdn: 'https://esm.sh/'
            }),
          ],
        }),
```
how to import uno.css in create-react-app, when i use below
```
import 'virtual:uno.css'
```
the error message is 
```
Module build failed: UnhandledSchemeError: Reading from "virtual:uno.css" is not handled by plugins (Unhandled scheme).
Webpack supports "data:" and "file:" URIs by default.
```

anyone can provide a creat-react-app use demo?

**Comment by zyyv** [comment]
Why not try use **vite** to create react app.😅 And check our [React Example](https://github.com/unocss/unocss/tree/main/examples/vite-react).

**Reply by wangfengyuan** [reply]
yes i saw this, but it's a old project，i mainly want to use custom icons auto created by svg
